dao/src/main/java/com/epam/esm/dao/seters/CertificateTagButchInsertPreparedStatementSetter.java

```diff
@@ -13,8 +12,8 @@ public class CertificateTagButchInsertPreparedStatementSetter implements BatchPr
     Integer[] tagIds;
     int certificateId;
 
-    public CertificateTagButchInsertPreparedStatementSetter(int certificateId, Set<Integer> tagIds){
-        if(isNull(tagIds) || tagIds.isEmpty()){
+    public CertificateTagButchInsertPreparedStatementSetter(int certificateId, Set<Integer> tagIds) {
+        if (isNull(tagIds) || tagIds.isEmpty()) {
```

Comment:
use org.apache.commons.collections4.CollectionUtils

Reply:
Done. Probably org.springframework.util.CollectionUtils can be used instead? There is nothing like 'isNotNull(Collection<?>)' there, but it is a part of spring core framework functionality, therefore, there will be no need to add new dependency to the module for one utility method only.